### ISSUE TITLE
"Something went wrong" error message in webapp

### ISSUE THREAD

**Issue by marcobergman** [meri-imperiumi/signalk-logbook#27]
This only happens after I have entered the first log with, admittedly, hardly any data. When I have no file sailsconfiguration.json, I get the message below in the server log; when I have configured at least one sail; the message below is no longer given, but I still get Something went wrong. Deleting all files from plugin-config-data/signalk-logbook makes the message go away.

Signal K Server version 1.46.3 on a raspberry 3

```
pi@openplotter:~/.signalk/plugin-config-data $ cat signalk-logbook/2023-03-07.yml
- datetime: 2023-03-07T19:00:42.145Z
  text: tralala
  author: pi
  crewNames:
    - Marco
  category: navigation
```

`Mar 07 19:04:14 TypeError: Cannot read properties of undefined (reading 'sails') at /home/pi/.signalk/node_modules/@signalk/sailsconfiguration/index.js:203:36 at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:95:5) at next (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/route.js:144:13) at Route.dispatch (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/route.js:114:3) at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:95:5) at /usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:284:15 at Function.process_params (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:346:12) at next (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:280:10) at Function.handle (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:175:3) at router (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:47:12) at Layer.handle [as handle_request] (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/layer.js:95:5) at trim_prefix (/usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:328:13) at /usr/lib/node_modules/signalk-server/node_modules/express/lib/router/index.js:286:9 at Function.proces`

**Comment by bergie** [comment]
Thanks! Looks like the issue is with the sailsconfiguration plugin but gets triggered by logbook. I’ll have a look.

**Comment by bergie** [comment]
Fix is released as 0.4.2. The sailsconfiguration issue is actually unrelated, but there is a fix in PR signal/sailsconfiguration#9
